src/Utils/filterByLatest.js

```diff
@@ -0,0 +1,13 @@
+export const filterByLatest = (filteredArray, byLatest) => {
+  const withouFiltered = [...filteredArray];
+  if (byLatest === "") {
+    return withouFiltered;
+  } else {
+    withouFiltered.sort((a, b) => {
+      let date1 = a.snippet.publishedAt.slice(0, 10).split("-").join("");
```

Comment:
you could extract date using a function instead of using split,slice,join

Reply:
Noted, will fix this.